### PR TITLE
trufflehog: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/common/trufflehog.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Escanea un repositorio Git en busca de secretos verificados:
+
+`trufflehog git {{https://github.com/trufflesecurity/test_keys}} --only-verified`
+
+- Escanea una organización GitHub en busca de secretos verificados:
+
+`trufflehog github --org={{trufflesecurity}} --only-verified`
+
+- Escanea un repositorio de GitHub en busca de claves verificadas y generar un archivo de salida JSON:
```
Comment (tricantivu) [inline pages.es/common/trufflehog.md:14]:
```suggestion
- Escanea un repositorio de GitHub en busca de claves verificadas y genera un archivo de salida JSON:
```

File: pages.es/common/trufflehog.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Escanea cubos S3 utilizando roles IAM:
+
+`trufflehog s3 --role-arn={{iam-role-arn}}`
+
+- Escanea archivos o directorios individuales:
+
+`trufflehog filesystem {{ruta/al/archivo_o_directorio1 ruta/al/archivo_o_directorio2 ...}}`
+
+- Escanea una imagen Docker en busca de secretos verificados:
```
Comment (tricantivu) [inline pages.es/common/trufflehog.md:34]:
```suggestion
- Escanea una imagen de Docker en busca de secretos verificados:
```

File: pages.es/common/trufflehog.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# trufflehog
+
+> Encuentra y verifica credenciales en archivos, repositorios Git, cubos S3 e imágenes Docker.
```
Comment (tricantivu) [inline pages.es/common/trufflehog.md:3]:
```suggestion
> Encuentra y verifica credenciales en archivos, repositorios de Git, cubos S3 e imágenes Docker.
```

File: pages.es/common/trufflehog.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+> Más información: <https://github.com/trufflesecurity/trufflehog>.
+
+- Escanea un repositorio Git en busca de secretos verificados:
```
Comment (tricantivu) [inline pages.es/common/trufflehog.md:6]:
```suggestion
- Escanea un repositorio de Git en busca de secretos verificados:
```

File: pages.es/common/trufflehog.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`trufflehog git {{https://github.com/trufflesecurity/test_keys}} --only-verified`
+
+- Escanea una organización GitHub en busca de secretos verificados:
```
Comment (tricantivu) [inline pages.es/common/trufflehog.md:10]:
```suggestion
- Escanea una organización de GitHub en busca de secretos verificados:
```

File: pages.es/common/trufflehog.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`trufflehog github --org={{trufflesecurity}} --only-verified`
+
+- Escanea un repositorio de GitHub en busca de claves verificadas y generar un archivo de salida JSON:
+
+`trufflehog git {{https://github.com/trufflesecurity/test_keys}} --only-verified --json`
+
+- Escanea un repositorio GitHub junto con sus incidencias y solicitudes de extracción:
```
Comment (tricantivu) [inline pages.es/common/trufflehog.md:18]:
```suggestion
- Escanea un repositorio de GitHub junto con sus incidencias y solicitudes de extracción:
```

**Review by tricantivu — APPROVED** [review]

**Pull request merged**